src/services/utils/date-helpers.js

```diff
@@ -0,0 +1,31 @@
+import moment from 'moment';
+
```

Comment:
These kind of helpers are what `src/scripts` is for (yes it is TERRIBLY named), so this should probably be there instead of a new `src/servies/utils` directory.

On the other hand, all this logic is extremely specific to the `WebMonitoringDb` class. Is there a good reason to separate it into its own module?